velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -52,5 +53,10 @@ int main(int argc, char** argv) {
       "replace",
       "might_contain",
       "unix_timestamp"};
-  return FuzzerRunner::run(FLAGS_seed, skipFunctions);
+
+  // Required by spark_partition_id function.
+  std::unordered_map<std::string, std::string> queryConfigs = {
+      {facebook::velox::core::QueryConfig::kSparkPartitionId, "0"}};
```

Comment:
nit: perhaps, use some "interesting" value here instead of zero; maybe 123

Reply:
Upated, thanks.